lib/src/builtins/styled_element_builtin.dart

```diff
@@ -172,6 +172,11 @@ class StyledElementBuiltIn extends HtmlExtension {
           textDecoration: TextDecoration.lineThrough,
         );
         break;
+      case "details":
+        styledElement.style = Style(
+          display: Display.block,
+        );
+        break;
```

Comment:
This style needs to be added to `DetailsElementBuiltIn` in the `prepare` method, rather than this class.

Reply:
You're sort of right in the StyledElementBuiltin it indeed doesn't do anything. However if I add it into the DetailsElementBuiltin it causes different rendering issues. I removed it for Summary and left it in just for Details; this seems to work properly.